src/pages/Overview/OverviewPage.tsx

```diff
@@ -387,10 +413,81 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
     );
   };
 
+  show = (showType: Show, namespace: string, graphType: string) => {
+    let destination = '';
+    switch (showType) {
+      case Show.GRAPH:
+        destination = `/graph/namespaces?namespaces=${namespace}&graphType=${graphType}`;
+        break;
+      case Show.APPLICATIONS:
+        destination = `/${Paths.APPLICATIONS}?namespaces=` + namespace;
+        break;
+      case Show.WORKLOADS:
+        destination = `/${Paths.WORKLOADS}?namespaces=` + namespace;
+        break;
+      case Show.SERVICES:
+        destination = `/${Paths.SERVICES}?namespaces=` + namespace;
+        break;
+      case Show.ISTIO_CONFIG:
+        destination = `/${Paths.ISTIO}?namespaces=` + namespace;
+        break;
+      default:
+      // Nothing to do on default case
+    }
+    history.push(destination);
+  };
+
+  showActions = (): any[] => {
+    const namespaceActions: any[] = [
```

Comment:
You created an exported type `OverviewNamespaceAction` somewhere else, isn't it this type that is used here ; and for  the other "any" here and there?